scopes/compositions/compositions/compositions.tsx

```diff
@@ -78,7 +78,7 @@ export function Compositions({ menuBarWidgets, emptyState }: CompositionsProp) {
             emptyState={emptyState}
             component={component}
             selected={selected}
-            queryParams={queryParams}
+            queryParams={`name=${queryParams}`}
```

Comment:
are we sure that `queryParams` will only contain the composition name? or could it contain other params as well?
maybe better to handle the name separately from the other params